src/DevChatter.DevStreams.Infra.Dapper/DapperCrudRepository.cs

```diff
@@ -86,6 +86,36 @@ public async Task<List<T>> GetAll<T>(string filter, string orderBy, object args)
             }
         }
 
+        /// <summary>
+        /// Returns all the channel data for a channel object.
+        /// </summary>
+        /// <returns>All Channel Info in a List<Channel> Object </Channel></returns>
+        public async Task<List<Channel>> GetAllChannelInfo()
```

Comment:
Check out the `IChannelAggregateService`, which should be able to get you the same information as this. You can use that instead of creating your own method.

Reply:
Had to create my own method, but it is in the new location, good? Or change again?